mlir/lib/Dialect/MIOpen/Transforms/AffixTuningParameters.cpp

```diff
@@ -168,6 +166,11 @@ void AffixTuningParameters::affixTuningParametersImpl(T &op) {
     }
   };
 
+  std::string perfConfig;
+  if (auto perfConfigAttr =
+          op->template getAttrOfType<StringAttr>("perf_config")) {
+    perfConfig = perfConfigAttr.getValue().str();
+  }
   auto xdlopsV2Attr = op->template getAttrOfType<BoolAttr>("xdlopsV2");
   auto ignoreTuningAttr = op->template getAttrOfType<BoolAttr>("ignore_tuning");
   bool ignoreTuning =
```

Comment:
You can get rid of this, as well as line 188.

Reply:
should be cleaned up now.